covalent/_dispatcher_plugins/local.py

```diff
@@ -73,19 +77,23 @@ def wrapper(*args, **kwargs) -> str:
             Returns:
                 The dispatch id of the workflow.
             """
+            if isinstance(orig_lattice, types.FunctionType):
+                app_log.error(
+                    "Dispatcher object must be covalent workflow. Instead, a function was passed"
```

Comment:
Once above is addressed should be:

`Electron was attempted to be dispatched instead of Lattice, please check that your lattice decorator has @lattice then @electron specified in that order not @electron then @lattice.`